neuralpredictors/layers/readouts/__init__.py

```diff
@@ -0,0 +1,28 @@
+from .point_pooled import *
+from .gaussian import *
+from .factorized import *
+from .attention import *
+from .pyramid import *
+from .multi_readout import *
```

Comment:
I would suggest using the actual names of the things you want to import instead of using `*`. This way of doing things might lead to hard to detect bugs in the future. For example it would be possible to import the same name from different modules without knowing.